### PR TITLE
Raise exception if no plugin is run

### PR DESCRIPTION
**Pull request by bjoernricks**
**What**:

Fail if no plugin is run.

**Why**:

This is very likely an error on the users side for example a typo in the
include parameter.

**How**:

**Checklist**:

- [x] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

File: tests/test_runner.py [hunk 1]
```diff
@@ -247,7 +249,19 @@ def test_runner_run_ok_without_debug(self):
 
         output = f.getvalue()
         self.assertIn(f"Checking {nasl_file}", output)
-        self.assertNotIn("Running plugin check_missing_desc_exit", output)
+        self.assertNotIn("Results for plugin check_missing_desc_exit", output)
```
Comment (y0urself) [inline tests/test_runner.py:252]: Why? The output is always 
```
ℹ Checking common2/2008/gb_w3c_amaya_mult_bof_vuln_dec08_lin.nasl (413/415)
ℹ     Running plugin check_openvas_lint
×         openvas-nasl-lint not found within your PATH. Please install 'openvas' and make sure it is available within your PATH.
ℹ     Running plugin check_updated_date_version
×         Changed VT has a not updated script_version();
```
Or not?

```suggestion
        self.assertNotIn("Running plugin check_missing_desc_exit", output)
```

**Review by y0urself — CHANGES_REQUESTED** [review]

File: tests/test_runner.py (hunk 1 above)
Comment (bjoernricks) [reply tests/test_runner.py:252]: Because actually the plugin has run already and therefore the statement is technically wrong

Comment (y0urself) [reply tests/test_runner.py:252]: Yes that is correct. But it is not enough to change this in a single assertion within a test. :P 
Should we change it all over, or leave it?

Comment (bjoernricks) [reply tests/test_runner.py:252]: Oh sorry this change should not have been included here. I'll fix that.

**Review by y0urself — APPROVED** [review]

**Pull request merged**